app/assets/javascripts/stores/settings_store.js

```diff
@@ -0,0 +1,10 @@
+import StockStore from './stock_store.js';
+
+const settingsStore = new StockStore({
```

Comment:
Please use Redux for this; we're trying to migrate away from the Flux stores (including all the ones that use StockStore).

Reply:
Nevermind... looks like you're not actually using this file, and already have it hooked up for Redux.